tests/json.rs

```diff
@@ -243,6 +247,52 @@ fn dependency_argument() {
         }
       },
       "settings": {
+        "allow_duplicate_recipes": false,
+        "dotenv_load": null,
+        "export": false,
+        "positional_arguments": false,
+        "shell": null,
+        "windows_powershell": false,
+      },
+      "warnings": [],
+    }),
+  );
+}
+
+#[test]
+fn duplicate_recipes() {
+  test(
+    "
+      set allow-duplicate-recipes
+      alias f := foo
+
+      foo:
+      foo:
```

Comment:
This test is slightly improved if the two `foo`s are different, so you can see the correct one in the output:

```suggestion
      foo:
      foo bar:
```